homeassistant/components/trafikverket_train/sensor.py

```diff
@@ -167,16 +167,26 @@ def extra_state_attributes(self):
             deviations = ", ".join(state.deviations)
         if self._delay_in_minutes is not None:
             self._delay_in_minutes = self._delay_in_minutes.total_seconds() / 60
-        return {
+        attributelist = {
             ATTR_DEPARTURE_STATE: self._departure_state,
             ATTR_CANCELED: state.canceled,
             ATTR_DELAY_TIME: self._delay_in_minutes,
-            ATTR_PLANNED_TIME: state.advertised_time_at_location,
-            ATTR_ESTIMATED_TIME: state.estimated_time_at_location,
-            ATTR_ACTUAL_TIME: state.time_at_location,
             ATTR_OTHER_INFORMATION: other_information,
             ATTR_DEVIATIONS: deviations,
         }
+        if state.advertised_time_at_location:
+            attributelist[
+                ATTR_PLANNED_TIME
+            ] = state.advertised_time_at_location.astimezone(self._timezone)
```

Comment:
We should convert it to UTC time and isoformat string.

https://github.com/home-assistant/core/blob/5e25df91b232798cb43a08cc9baa27edfa0b286e/homeassistant/util/dt.py#L67-L68

https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat